packages/components/src/Description/Description.stories.tsx

```diff
@@ -87,6 +151,21 @@ export const JustifyContent = () => {
   );
 };
 
+export const JustifyContentCanCopy = () => {
```

Comment:
```suggestion
export const JustifyContentWithCanCopy = () => {
```

либо можно показать этот пример в рамках  JustifyContent, тут на твое усмотрение